GitUI/CommandsDialogs/RepoHosting/ForkAndCloneForm.Designer.cs

```diff
@@ -501,6 +517,28 @@ private void InitializeComponent()
             this.destinationTB.TextChanged += new System.EventHandler(this._destinationTB_TextChanged);
             this.destinationTB.Validating += new System.ComponentModel.CancelEventHandler(this._destinationTB_Validating);
             // 
+            // depthUpDown
+            // 
+            this.depthUpDown.Location = new System.Drawing.Point(10, 151);
+            this.depthUpDown.Name = "depthUpDown";
+            this.depthUpDown.Value = new decimal(new int[] {
+            0,
+            0,
+            0,
+            0});
+            this.depthUpDown.Minimum = new decimal(new int[] {
+            0,
+            0,
+            0,
+            0});
+            this.depthUpDown.Maximum = new decimal(new int[] {
+            999,
+            0,
+            0,
+            0});
+            this.depthUpDown.Size = new System.Drawing.Size(100, 20);
+            this.depthUpDown.TabIndex = 27;
```

Comment:
Can you please confirm the tab order is logical?

Reply:
To be honest, I just copied it. It's the same as any other usage of NumericUpDown in GE.